svthalia/concrexit
# link event to the album title and date

Closes #1488 



### Summary
When an album is created, it is possible to link an event and use it's title and date instead of filling it in yourself.
This also fixes the multilanguage field for title and it's migrations.

### How to test
Add an album without a title or date filled in and select an event
Save and check if the correct date and title get added

Add an album without a title, date or event, this should give an error message
Add an album without a title and without an event, this should give an error message
Add an album without a date and without an event, this should give an error message
